package.json

```diff
@@ -43,12 +43,13 @@
   "devDependencies": {
     "@hapi/code": "^8.0.2",
     "@hapi/lab": "^24.1.0",
+    "async": "3.2.0",
+    "lab-transform-typescript": "^3.0.1",
```

Comment:
Q: Should we freeze the version?

Reply:
yes, and validate with https://github.com/rjrodger/seneca-plugin-validator - work for later